triggers/runner/src/main/resources/logback.xml

```diff
@@ -13,7 +13,7 @@
     <logger name="daml.warnings" level="WARN" />
     <logger name="com.daml.lf.engine.trigger" level="DEBUG" />
 
-    <root level="INFO">
+    <root level="${LOG_LEVEL_ROOT:-INFO}">
```

Comment:
Preferring this manner for configuring log levels (over using `Logger.setLevel` in Scala-land) as it is more transparent to the end user